client/src/pages/Login.js

```diff
@@ -14,7 +14,7 @@ import CssBaseline from "@material-ui/core/CssBaseline";
 import Typography from "@material-ui/core/Typography";
 import MuiAlert from "@material-ui/lab/Alert";
 
-import { loginUser, setActiveUsers } from "../apis/user";
+import { loginUser } from "../apis/user";
```

Comment:
If you are not using `setActiveusers` here, I think you can delete the whole function from `apis/user.js`. 
(I just noticed that right after the signup, there are no friends. so don't need it)

Reply:
I'm using `setActiveUsers `in sidebar